helm-charts/sep-service/README.md

```diff
@@ -61,9 +61,75 @@ metadata:
 type: Opaque
 ```
 
+# Assets Configuration
```

Comment:
😎 Hey, this is cool. We were discussing of converting the `assets.json` to `assets.yaml`.